### DISCUSSION TITLE
PF version 5.3.21 "/" rendering as "\/" in watermark

### DISCUSSION THREAD

**Discussion by mikejones0925** [primefaces/community#369] [PrimeFaces] [answered]
In a watermark, we want the text to be (for example) "texta/textb"<img width="80" alt="image" src="https://github.com/primefaces/community/assets/49534755/9f8b995e-4c6a-48cf-a1d7-9c62cd31045c">
. This is being rendered as <img width="67" alt="image" src="https://github.com/primefaces/community/assets/49534755/0b816658-f796-41eb-9875-3fb1ba5a7ade">

. This is happening since updgrading to version 5.3.21

`
<f:facet name="filter">
  <p:inputText id="textaTextbFilter" onkeyup="PF('tableForms').filter()" styleClass="Fleft" style="width: 89%;">
    <p:watermark for="textaTextbFilter" value="texta/textb" />							
  </p:inputText>
</f:facet>
`

**Comment by melloware** [comment, marked as answer]
I don’t think you will get much help here on such an old version of PF.   And watermark was removed from PF in later versions because you can just use pass through attributes to add the placeholder.
